tsnet/example/tshello/tshello.go

```diff
@@ -25,10 +25,13 @@ var (
 func main() {
 	flag.Parse()
 	s := new(tsnet.Server)
+	s.Close()
```

Comment:
forgot the defer

Reply:
ping

Reply:
Apologies for the late reply @bradfitz! I was busy but now back home. I've made the update, please take a look!